CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/attribute/utilities/AttributeUtilities.java

```diff
@@ -48,7 +48,7 @@ private AttributeUtilities() {
     public static List<String> getAttributeNames(final GraphElementType graphElementType, String attributeDescription) {
         final List<String> attributeNames = new ArrayList<>();
         for (Attribute attribute : getAttributes(graphElementType)) {
-            if (attribute.getAttributeType().equals(attributeDescription) || attributeDescription.equals("")) {
+            if (attribute.getAttributeType().equals(attributeDescription) || "".equals(attributeDescription)) {
```

Comment:
`... || attributeDescription.isEmpty()`?

Reply:
What about `StringUtils.isEmpty(attributeDescription)` for the included null check